app/graphql/types/fees/object.rb

```diff
@@ -20,6 +20,12 @@ class Object < Types::BaseObject
 
       field :creditable_amount_cents, Integer, null: false
 
+      field :amount_currency, Types::CurrencyEnum, null: false
+
+      def amount_currency
+        object.currency
```

Comment:
Actually, this fields is already defined in `/app/graphql/types/invoices/invoice_item.rb` which is implemented by this object

Reply:
indeed, removed